### PR TITLE
feat: make menu structure more consistent

### PR DESCRIPTION
**Pull request by hacdias**
This removes the `Install` navbar item, which now links to a non-existing section, and it also updates the `About` link to just link to the website without an anchor link.

Alternatively, I would suggest re-adding the anchor links to the headers on the ipfs.tech website and then updating them here accordingly. For now, I think it's just better to clean this up in the blog.

### CODE REVIEW

**Comment by cwaring** [comment]
Agree, some cleanup is needed for the blog now (still in the backlog). I suggest we mirror the menu structure for now, also adding the community page + developers link and refine from there later.

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -42,16 +42,18 @@ const themeConfigDefaults = {
...
   headerLinks: [
-    { text: 'About', link: 'https://ipfs.tech/#why' },
-    { text: 'Install', link: 'https://ipfs.tech/#install' },
+    { text: 'About', link: 'https://ipfs.tech/' },
+    { text: 'Community', link: 'https://ipfs.tech/community/' },
+    { text: 'Developers', link: 'https://ipfs.tech/developers/' },
     { text: 'Docs', link: 'https://docs.ipfs.tech/' },
     { text: 'Team', link: 'https://ipfs.tech/team' },
     { text: 'Blog', link: '/' },
     { text: 'Help', link: 'https://ipfs.tech/help' },
```
Comment (hacdias) [inline src/.vuepress/config.js:51]: @cwaring for mirroring: should we remove `Team` and `Help`? I can't even find links for this in ipfs.tech, even though the links work.

Comment (cwaring) [reply src/.vuepress/config.js:51]: Yes that would be best, these aren't maintained at this stage so they have been soft deprecated. We may introduce them back via the footer nav in the future so the pages remain active for existing links in the meantime.

Comment (hacdias) [reply src/.vuepress/config.js]: @cwaring done! Feel free to merge.

**Review by cwaring — APPROVED** [review]

**Comment by hacdias** [comment]
Do you have any idea what's going on with the Fleek build?

**Comment by cwaring** [comment]
It seems like their GH actions are playing up atm, we have an issue open with them but it's still unresolved. We can merge and deploy manually for now if it doesn't trigger.

cc @vesahc

**Pull request merged**